custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -511,7 +511,7 @@ def _reg_args_unstructured_mode(*parsers):
 
     @staticmethod
     def _reg_arg_target_type(*parsers):
-        target_types = [e.value for e in TargetType]
+        target_types = [e.value for e in TargetType if e != TargetType.CLASSIFICATION]
```

Comment:
Why classification is on included?

Reply:
Oh, got it

Reply:
let's add `TargetType.ALL` so that we don't have to add `if e != TargetType.CLASSIFICATION`

Reply:
@rvorobii that won't work and isn't necessary. `TargetType` is an actual `Enum` type, so it's already iterable over its values.  The issue here was that it since `CLASSIFICATION` is one of the enum values, the help message had `regression, ..., multiclass, binary, [multiclass, binary]`

Reply:
oh. I see what you mean.

Reply:
I understand the problem.
the `ALL` I thiought about would've contain `regression`, `multiclass`, `binary`